PowerShell/ScubaGear/Testing/Unit/Rego/EXO/EXOConfig_06_test.rego

```diff
@@ -9,33 +9,15 @@ import data.utils.key.PASS
 # Policy MS.EXO.6.1v1
 #--
 test_Domains_Contacts_Correct if {
-    Output := exo.tests with input as {
-        "sharing_policy": [
-            {
-                "Domains": [
-                    "domain1",
-                    "domain2"
-                ],
-                "Name": "A"
-            }
-        ]
-    }
+    Output := exo.tests with input.sharing_policy as [SharingPolicy]
 
     TestResult("MS.EXO.6.1v1", Output, PASS, true) == true
 }
 
 test_Domains_Contacts_Incorrect if {
-    Output := exo.tests with input as {
-        "sharing_policy": [
-            {
-                "Domains": [
-                    "*:ContactsSharing",
-                    "domain1:CalendarSharingFreeBusyDetail"
-                ],
-                "Name": "A"
-            }
-        ]
-    }
+    SharingPolicy1 := json.patch(SharingPolicy, [{"op": "add", "path": "Domains", "value":["*:ContactsSharing", "domain1:CalendarSharingFreeBusyDetail"]}])
```

Comment:
I know this is just a warning in Regal not an error, still, I'd recommend breaking up this line to get rid of the "line too long" warning. Something like this should work:
```
SharingPolicy1 := json.patch(SharingPolicy, [{
    "op": "add",
    "path": "Domains",
    "value": ["*:ContactsSharing", "domain1:CalendarSharingFreeBusyDetail"]
}])
```